NEWS.md

```diff
@@ -443,6 +443,10 @@ Deprecated or removed
     this is not true. If you are certain you need the old behavior, it is temporarily available
     as `Base._isleaftype` ([#17086]).
 
+  * Constructors for `LibGit2.UserPasswordCredentials` and `LibGit2.SSHCredentials` no longer take a
+    `prompt_if_incorrect` argument. Controlling prompting behavior has been moved to
+    `LibGit2.CredentialPayload`.
```

Comment:
Might want to also clarify a few points:

1. `prompt_if_incorrect` has also been renamed to `allow_prompt` in the `CredentialPayload`
2. `use_ssh_agent` is now a Bool vs `Char`.

Reply:
I'll update the news to mention `allow_prompt`. As for change to `use_ssh_agent` in `CredentialPayload` I'll avoid mentioning that since the entire `CredentialPayload` is new to 0.7.